.vscode/extensions.json

```diff
@@ -1,3 +1,3 @@
 {
-  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode", "orta.vscode-jest"]
+  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode"]
```

Comment:
Should we add `vitest.explorer` into the list?

Reply:
Yeah good call.